masterlist.yaml

```
@@ -6006,3 +6006,11 @@ plugins:
     clean:
       - crc: 0x489330C5
         util: 'FO4Edit v4.0.3h (Hotfix 1)'
+
+  - name: 'ARMOR-WEIGHT-OMNITOBW-PATCH.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/54173/' ]
+    req: [ 'Unofficial Fallout 4 Patch.esp' ]
+    req: [ 'ArmorWeight.esp' ]
```

For multiple items in `req:`, you have to format it this way.
```suggestion
    req:
      - 'ArmorWeight.esp'
      - 'Unofficial Fallout 4 Patch.esp'
```